drivers/lora/sx1276.c

```diff
@@ -84,13 +82,12 @@ void SX1276SetAntSw(u8_t opMode)
 void SX1276Reset(void)
 {
 	gpio_pin_configure(dev_data.reset, GPIO_RESET_PIN,
```

Comment:
The pin is configured inside `sx1276_lora_init` function. Here we could simply call `gpio_pin_set`.

Reply:
I thought about that, but there's no documentation on why this function is present.  It could be called before the driver is initialized.  So I left it as originally implemented.

Reply:
There are no in tree users, so it may be only the application. In which case it will be called after the `sx1276_lora_init` function. In fact, this looks like dead code as it doesn't seem to be present in any of the header files.